drivers/flash/spi_nor.h

```diff
@@ -61,6 +61,12 @@
 #define SPI_NOR_CMD_PP_1_1_4_4B  0x34  /* Quad Page program (1-1-4) 4 Byte Address */
 #define SPI_NOR_CMD_PP_1_4_4_4B  0x3e  /* Quad Page program (1-4-4) 4 Byte Address */
 
+#define SPI_NOR_CMD_WR_WRARG 0x71 /* Write Any Register for S28Hx512T */
+#define SPI_NOR_CMD_RSFDPID  0x5A /* Read SFDP ID for S28Hx512T */
+#define SPI_NOR_CMD_RREG     0x65 /* Read Any Register for S28Hx512T */
+#define SPI_NOR_CMD_SE_256KB 0xDC /* Sector Erase 256KB for S28Hx512T */
+#define SPI_NOR_CMD_ERCHP    0x60 /* Erase Chip for S28Hx512T */
+
```

Comment:
No. it fit his is S28Hx512T specific, it does not land here. Get your own header for that. Specifically that nothing of this is used in spi_nor.c